scalafmt-core/shared/src/main/scala/org/scalafmt/rewrite/CamelCaseVariableNaming.scala

```diff
@@ -0,0 +1,40 @@
+package org.scalafmt.rewrite
+
+import scala.meta.{Pat, Tree}
+
+object CamelCaseVariableNaming extends Rewrite {
+  override def create(implicit ctx: RewriteCtx): RewriteSession =
+    new CamelCaseVariableNaming
+}
+
+class CamelCaseVariableNaming(implicit ctx: RewriteCtx) extends RewriteSession {
+
+  def toCamelCaseNaming(str: String): String = {
```

Comment:
Variable names in Scala may start with underscore:
```scala
val _abc = 1
```

And this function works wrong on it:
```
@ toCamelCaseNaming("_abc")
res2: String = "Abc"
```